File: src/Exception/DivisionByZeroException.java
package Exception;

/**
 * Created by vladc on 28.10.2016.
 */
public class DivisionByZeroException extends ExpressionEvaluationException {
    public DivisionByZeroException() {
        super();
    }
    public DivisionByZeroException(String message) {
        super(message);
    }
}

File: Toy-Language-Interpreter/MyException/DivisionByZeroException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Toy_Language_Interpreter.MyException
{
    class DivisionByZeroException : ExpressionEvaluationException
    {
        public DivisionByZeroException() { }
        public DivisionByZeroException(string msg) : base(msg) { }
        public DivisionByZeroException(string msg, Exception e) : base(msg, e) { }
    }
}
